onnxruntime/core/providers/migraphx/migraphx_execution_provider_utils.h

```diff
@@ -134,6 +134,11 @@ inline bool canEvalNodeArgument(const GraphViewer& graph,
       continue;
     }
 
+    const auto& optype = node->OpType();
```

Comment:
We shouldn't be adding a check here for operators as we're checking node arguments in a general sense. I think the way around this is to remove stuff here and try the inference